photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/kernels/StationaryKernel.scala

```diff
@@ -56,6 +58,20 @@ abstract class StationaryKernel(
    */
   protected[kernels] def fromPairwiseDistances(dists: DenseMatrix[Double]): DenseMatrix[Double]
 
+  /**
+   * Unwraps hyper-parameters from scaled values to true values
+   *
+   * @param x the matrix of points, where each of the m rows is a point in the space and some columns contain scaled values
+   * @return the unwrapped x that all the columns have true values
+   */
+  protected[kernels] def transform(x: DenseMatrix[Double]): DenseMatrix[Double] = {
+    val u = x.copy
+    indexedTransformMap.map { case (index: Int, transform: (Double => Double)) =>
```

Comment:
Let's use a different name for the function here. It might cause some reading confusion, since the parent function is also called `transform`.